src/components/tools/tool-tile.tsx

```diff
@@ -68,7 +56,7 @@ interface IToolTileBaseProps {
   docId: string;  // ephemeral contentId for the DocumentContent
   documentContent: HTMLElement | null;
   isUserResizable: boolean;
-  scale?: number;
+  scale?: number; // This isn't used anymore as far as I can tell
```

Comment:
```suggestion
  scale?: number;
```
It may not be used in this component any more, but it is spread to the individual components that do still use it to scale their rendering and event handling, so this comment is unnecessary.

Reply:
Oops, I missed that. I keep forgetting the all the props to the ToolTileComponent get passed down to all of the tool components.

Reply:
fixed, and I added a comment on `IToolTileBaseProps` to help me remember how it is used.